### ISSUE TITLE
manage_edd_commissions_custom_column debug data hook issue

### ISSUE THREAD

**Issue by davidsherlock** [easydigitaldownloads/EDD-Commissions#207]
I have written a custom invoicing system for the commissions plugin, however there is one issue with the above hook:

Within EDD_C_List_Table.php, on line 101 we currently have this code:

```
default:
return print_r( $item, true ); //Show the whole array for troubleshooting purposes
```

This is resulting in my hook spitting out the following on the table:

![screenshot 2017-03-27 18 19 32](https://cloud.githubusercontent.com/assets/22668877/24369176/0c37e222-131b-11e7-9a23-3a68494b987a.png)

Commenting out the `return` line fixes the issue and shows my links correctly. Can this be commented out in the next release please? it's causing issues when new columns are added.

**Comment by rubengc** [comment]
I experienced the same

in PR #172 those lines was removed
https://github.com/easydigitaldownloads/EDD-Commissions/pull/172/commits/8736185ceab521f9657117ada443fcd82ce1623d#diff-8d6426379b1d7288f3f96fdfc67b8165

**Comment by pippinsplugins** [comment]
Whoops!

**Comment by pippinsplugins** [comment]
We'll get it fixed in 3.3.3
